ambry-router/src/test/java/com/github/ambry/router/InMemoryCloudDestinationErrorSimulationTest.java

```diff
@@ -0,0 +1,387 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.router;
```

Comment:
shouldn't this test class be placed under ambry.cloud, instead of ambry.router. The LatchBasedInMemoryCloudDestination is placed under ambry.cloud.

Reply:
After changed the test code to use networkClient.sendAndPoll instead of router, moved to cloud folder.